bot.py

```diff
@@ -3,32 +3,34 @@
 from discord_slash import SlashCommand
 from discord_slash.utils.manage_commands import create_permission
 from discord_slash.model import SlashCommandPermissionType
-import os
+from utils import get_env_vars
 from messages import commands
 
+server_id, role_id, bot_token = get_env_vars()
+
 client = discord.Client(intents=discord.Intents.default())
 slash = SlashCommand(client, sync_commands=True) # Declares slash commands through the client.
-guild_ids = [703325111209427015, 763798356484161566] # Test Server, Stellar Quest Server
+
 
 quest_roles = [
-	create_permission(766768688342499390, SlashCommandPermissionType.ROLE, True), # Lumenaut
-	create_permission(763798356484161566, SlashCommandPermissionType.ROLE, False),  # Everyone
+	create_permission(role_id, SlashCommandPermissionType.ROLE, True), # Lumenaut | Custom Role with Permissions from env var
+	create_permission(server_id, SlashCommandPermissionType.ROLE, False),  # Everyone | everyone role ID = Server ID 
 ]
 
 @client.event
 async def on_ready():
     print("Ready!")
 
-@slash.slash(name="ping", guild_ids=guild_ids)
-@slash.permission(guild_id=763798356484161566,permissions=quest_roles)
+@slash.slash(name="ping", guild_ids=[server_id]) # Set Server ID | This still needs to be a list, even though there's just one server
```

Comment:
Maybe include the possibility it provide multiple servers via environment variable?